tools/llvm-kompile-codegen/main.cpp

```diff
@@ -239,12 +242,14 @@ int main(int argc, char **argv) {
     finalize_debug_info();
   }
 
-  do_bitcode_linking(*mod);
+  do_bitcode_linking(*mod, (char *)alloc_cpp_o_ll, alloc_cpp_o_ll_len);
 
   if (!no_optimize) {
     apply_kllvm_opt_passes(*mod, hidden_visibility);
   }
 
+  do_bitcode_linking(*mod, (char *)opaque_ll, opaque_ll_len);
```

Comment:
looks like you forgot to call apply_inline_pass here

Reply:
Right, thank you. Addressed.